package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "bedrock-vc-status-list-context",
+  "version": "1.0.0",
+  "description": "Bedrock VC Status List 2021 Context (Bedrock Module)",
+  "main": "lib",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digitalbazaar/bedrock-vc-status-list-context.git"
+  },
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-vc-status-list-context/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-vc-status-list-context#readme",
+  "dependencies": {
+    "vc-status-list-context": "~1.0.0"
```

Is this intentional -- to only accept patches and not minor upgrades?

```suggestion
    "vc-status-list-context": "^1.0.0"
```

---

Fixed a92f012